packages/map-template/src/components/Directions/Directions.jsx

```diff
@@ -29,7 +30,7 @@ function Directions({ isOpen, onBack, directions }) {
     const [totalDistance, setTotalDistance] = useState();
     const [totalTime, setTotalTime] = useState();
 
-    const mapsIndoorsInstance = useContext(MapsIndoorsContext);
+    const mapsIndoorsInstance = useRecoilValue(mapsIndoorsInstanceState);
```

Comment:
Just to make sure I understand correctly the distinction between the `useRecoilState` and `useRecoilValue` - do we use the value here because of the usage of `dangerouslyAllowMutability`?

Reply:
Not quite. We use useRecoilValue when we know we should never update the state from within this component.